pkg/network/http/http_statkeeper.go

```diff
@@ -75,9 +77,22 @@ func (h *httpStatKeeper) add(tx *httpTX) {
 		atomic.AddInt64(&h.telemetry.malformed, 1)
 		return
 	}
-	path, rejected := h.processHTTPPath(rawPath)
+
+	path, rejected := h.processHTTPPath(tx, rawPath)
 	if rejected {
-		atomic.AddInt64(&h.telemetry.rejected, 1)
+		return
+	}
+
+	if Method(tx.request_method) == MethodUnknown {
+		atomic.AddInt64(&h.telemetry.malformed, 1)
+		log.Warnf("method should never be unknown: %s", tx.String())
```

Comment:
We should probably use something like `pkg/process/util.LogLimit` to make sure we don't flood the logs in case of a problem.